src/detections/configs.rs

```diff
@@ -1018,8 +1018,8 @@ pub struct OutputOption {
     pub exclude_status: Option<Vec<String>>,
 
     /// Only load rules with specific tags (ex: attack.execution,attack.discovery)
-    #[arg(help_heading = Some("Filtering"), long = "tags", value_name = "TAGS", use_value_delimiter = true, value_delimiter = ',', display_order = 460)]
-    pub tags: Option<Vec<String>>,
+    #[arg(help_heading = Some("Filtering"), long = "include_tags", value_name = "TAGS", conflicts_with = "exclude_tags", use_value_delimiter = true, value_delimiter = ',', display_order = 351)]
```

Comment:
```suggestion
    #[arg(help_heading = Some("Filtering"), long = "include-tags", value_name = "TAGS", conflicts_with = "exclude_tags", use_value_delimiter = true, value_delimiter = ',', display_order = 351)]
```

Reply:
@hitenkoku 
Thank you for checking!
How about changing only this? (does not change `conflicts_with`)

Reply:
@fukusuket Thank you for your suggest.
I apologize for the incorrect description on my part.
The content is indeed correct. I have adopted your suggestion.